Assets/Mirror/Runtime/NetworkIdentity.cs

```diff
@@ -500,7 +549,7 @@ void OnDestroy()
             }
         }
 
-        internal void OnStartServer()
+        internal void StartServer()
```

Comment:
Is this called anywhere?

Reply:
That is added as a listener in awake and would be called in a few places including NetworkServer.SpawnObject()